packages/utils/package.json

```diff
@@ -60,6 +60,9 @@
   "publishConfig": {
     "access": "public"
   },
+  "resolutions": {
+    "@types/react": "^17.0.48"
+  },
```

Comment:
Some dependency is pulling in `@types/react@*` which causes us to pull an incompatible version of `@types/react`, so let's force using v17.